modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -591,6 +591,21 @@ void ONNXImporter::populateNet(Net dstNet)
             }
             layerParams.set("num_output", layerParams.blobs[0].size[1] * layerParams.get<int>("group", 1));
             layerParams.set("bias_term", node_proto.input_size() == 3);
+
+            const DictValue* outShape = layerParams.ptr("output_shape");
+            if (outShape && outShape->size() == 4)
+            {
+                const int strideY = layerParams.get<int>("stride_h");
+                const int strideX = layerParams.get<int>("stride_w");
```

Comment:
Please use default values because `strides` might be optional:

```cpp
const int strideY = layerParams.get<int>("stride_h", 1);
const int strideX = layerParams.get<int>("stride_w", 1);
```

Reply:
This is done as well.